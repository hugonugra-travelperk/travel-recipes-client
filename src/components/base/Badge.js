import styled from "styled-components";

export const Badge = styled.span`
  padding: 0.3em 0.3em 0.3em 0.3em;
  margin-vertical: 2em;
  font-family: Quicksand, arial, sans-serif;
  border-radius: 1em;
  background: #ffc726;
  overflow: hidden;
`;
