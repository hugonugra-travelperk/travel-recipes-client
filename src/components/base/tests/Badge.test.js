import {render, cleanup} from "@testing-library/react"
import "@testing-library/jest-dom";
import {Badge} from "../Badge"

it("renders", () => {
  const {asFragment} = render(<Badge>Test text</Badge>);
  expect(asFragment()).toMatchSnapshot();
});
