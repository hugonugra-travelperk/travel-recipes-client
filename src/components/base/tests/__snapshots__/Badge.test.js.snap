// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <span
    class="sc-bdvvtL iEtpPF"
  >
    Test text
  </span>
</DocumentFragment>
`;
